src/OgGroupAudienceHelper.php

```diff
@@ -24,6 +24,15 @@ class OgGroupAudienceHelper {
    */
   const DEFAULT_FIELD = 'og_group_ref';
 
+  /**
+   * The name of the field which reference user to groups.
+   */
+  const USER_REFERENCE_FIELD = 'og_membership_reference';
```

Comment:
We still need this field because we still need a "field" on the user object to know to which groups the user belong to.